README.md

```diff
@@ -66,8 +71,39 @@ you need to wrap them in a stable tag to work around [this Glimmer issue](https:
 {{/react-panel}}
 ```
 
-Although this is possible, block form should be used as a tool to migrate Ember to React
-without the hard requirement to start with leaf components. It is highly recommended to have clean React component tree whenever possible for best performance.
+Although this is possible, block form should be used as a tool to migrate Ember
+to React without the hard requirement to start with leaf components. It is
+highly recommended to have clean React component tree whenever possible for best
+performance.
+
+## Using File Name Convention for React
```

Comment:
This section is the real content addition

Reply:
This seems like a good explanation of why, but doesn't really make it clear what ember-cli-react is providing in terms of file name support. I think it should make it clear that 1) you can name your jsx files in PascalCase and 2) either import them in templates using standard dasherized syntax or refer to them there using PascalCase with the aid of `react-component`.